README.md

```diff
@@ -1,24 +1,24 @@
-[![Build Status](https://travis-ci.org/adrcotfas/Goodtime.svg?branch=master)](https://travis-ci.org/goodtime-productivity/Goodtime) [![Crowdin](https://d322cqt584bo4o.cloudfront.net/goodtime/localized.svg)](https://crowdin.com/project/goodtime) 
+[![Build Status](https://travis-ci.org/adrcotfas/Goodtime.svg?branch=master)](https://travis-ci.org/goodtime-productivity/Goodtime) [![Crowdin](https://d322cqt584bo4o.cloudfront.net/goodtime/localized.svg)](https://crowdin.com/project/goodtime)
 
 # Goodtime
-
-<a href='https://play.google.com/store/apps/details?id=com.apps.adrcotfas.goodtime&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/images/apps/en-play-badge-border.png' height="80" /></a>
-<a href="https://f-droid.org/en/packages/com.apps.adrcotfas.goodtime">
-    <img src="http://i.imgur.com/baSPE7X.png"
-         alt="Get it on F-Droid" height="80">
-</a>
+[<img src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png"
+    alt="Get it on Google Play"
+    height="80">](https://play.google.com/store/apps/details?id=com.apps.adrcotfas.goodtime&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1)
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+    alt="Get it on F-Droid"
+    height="80">](https://f-droid.org/packages/com.apps.adrcotfas.goodtime)
 
 A minimalist but powerful productivity timer designed to keep you focused and free of distractions.
-<br>Alternate between focused work sessions and short breaks handled by the app.
-<br>Get rid of procrastination and improve your time management!
+Alternate between focused work sessions and short breaks, all handled by the app.
+Get rid of procrastination and improve your time management!
 
-<br>⏳ split your work into sessions of focus and then reward yourself with a break
-<br>☕ after more sessions of work, take a longer break
+⏳ split your work into focused sessions and then reward yourself with a break
+☕ after mulitple work sessions, take a longer break
```

Comment:
- typo in "multiple"
- we should have a new line here (two rows, each starting with the emoji)

Reply:
Thanks, missed the typo.
The markdown preview built in Github showed this on two lines, i'll check however.

Reply:
No, preview shows no line break. But now i've learnt how to break lines without creating a new paragraph: leave two spaces at the end of a line before entering carriage return :)